react-api-client/src/runs/useAddCapturePreviewImageToRun.ts

```diff
@@ -0,0 +1,58 @@
+import { useMutation, useQueryClient } from 'react-query'
+
+import { addCapturePreviewImageToRun } from '@opentrons/api-client'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type { UseMutateFunction, UseMutationResult } from 'react-query'
+import type {
+  CameraImageSettings,
+  DownloadedPreviewImageFileResponse,
+  ErrorResponse,
+} from '@opentrons/api-client'
+
+export interface AddCapturePreviewImageToRunParams {
+  runId: string
+  settings: CameraImageSettings
+}
+
+export type UseAddCapturePreviewImageToRunMutationResult = UseMutationResult<
+  DownloadedPreviewImageFileResponse,
+  AxiosError<ErrorResponse>,
+  AddCapturePreviewImageToRunParams
+> & {
+  addCapturePreviewImageToRun: UseMutateFunction<
+    DownloadedPreviewImageFileResponse,
+    AxiosError<ErrorResponse>,
+    AddCapturePreviewImageToRunParams
+  >
+}
+export function useCapturePreviewImageToRun(): UseAddCapturePreviewImageToRunMutationResult {
+  const host = useHost()
+  const queryClient = useQueryClient()
+
+  const mutation = useMutation<
+    DownloadedPreviewImageFileResponse,
+    AxiosError<ErrorResponse>,
+    AddCapturePreviewImageToRunParams
+  >(({ runId, settings }) =>
+    addCapturePreviewImageToRun(host!, runId, settings)
+      .then(response => {
+        queryClient
+          .invalidateQueries([host, 'runs', runId])
```

Comment:
```suggestion
          .invalidateQueries([host, 'runs', runId, 'camera', 'capturePreviewImage'])
```
I recommend spending some time reading the React Query docs on [query keys](https://tanstack.com/query/v5/docs/framework/react/guides/query-keys). These are pretty fundamental to how React Query works, and now's a good time to get familiar with them if you aren't already!

We don't want to force a refetch for all the run data if we can avoid it.